### ISSUE TITLE
Any luck changing LR scheduler type through config?

### ISSUE THREAD

**Issue by Fikxzer** [ostris/ai-toolkit#94]
Would like to know if anyone has successfully been able to switch the learning rate schedule. Ideally I would be able to switch to CosineAnnealingLR and change the T_max and eta_min value.

**Comment by Fikxzer** [comment]
I Figured out a work around.
in config file added the 3 lines below the lr value
        optimizer: "adamw8bit"
        lr: 1e-3
        lr_scheduler: "cosine"
        lr_scheduler_params:
          total_iters: 2000

and to get the eta_min set I changed the scheduler.py file adding the "eta_min=1e-4"

if name == "cosine":
        if 'total_iters' in kwargs:
            kwargs['T_max'] = kwargs.pop('total_iters')
        return torch.optim.lr_scheduler.CosineAnnealingLR(
            optimizer,eta_min=1e-4, **kwargs
